config/noi.json

```diff
@@ -0,0 +1,25 @@
+{
+    "feed_data": {
+        "system_information": {
+            "email": "info@carsharing.bz.it",
+            "feed_contact_email": "mail@leonard.io",
+            "language": "de",
+            "license_url": "https://www.govdata.de/dl-de/by-2-0",
+            "license_id": "DL-DE-BY-2.0",
```

Comment:
Is the license the appropriate one for an Italian feed?

Reply:
I have no idea and we will have to ask @rcavaliere.

When I click "API Licenses" on https://swagger.opendatahub.com I get a 404.

Reply:
This dataset has a CC0 license. This information is normally available on the page https://opendatahub.com/datasets/, but we did an update of this page recently and this information is no more shown (it's a mistake). We are integrating this information again

Reply:
Changed in 4592eef6b097dc70033ea867fffa8aad088c7a8d